.azure-pipelines/all.yml

```diff
@@ -17,27 +22,43 @@ jobs:
     matrix:
       Go113:
         GO_VERSION: "1.13"
-      GoTIP:
-        GO_VERSION: "tip"
   steps:
   - task: GoTool@0
     displayName: Use Golang $(GO_VERSION)
     inputs:
       version: $(GO_VERSION)
+      gobin: $(Agent.BuildDirectory)/gopath/bin
+      steps:
+  - task: Go@0
+    displayName: Get Go dependencies
+    inputs:
+      command: 'get'
+      arguments: "github.com/mattn/goveralls github.com/onsi/ginkgo github.com/onsi/gomega \
```

Comment:
```suggestion
      arguments: |
        github.com/mattn/goveralls \
        github.com/onsi/ginkgo \
        github.com/onsi/gomega \
        github.com/Masterminds/glide \
        golang.org/x/lint/golint
```

Reply:
Looks like the way `arguments` are stripped causes the whitespace around the go package to be removed, so it thinks its one giant package and fails. Updated it to look more readable though.